packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/WelcomeCard.tsx

```diff
@@ -1,29 +1,90 @@
 import React, { FC } from 'react';
 import { Trans } from 'react-i18next';
 import automationIcon from 'src/modules/Overview/images/automation.svg';
+import { DetailsItem } from 'src/modules/Providers/utils';
 import { useForkliftTranslation } from 'src/utils/i18n';
 
 import { V1beta1ForkliftController } from '@kubev2v/types';
-import { Card, CardBody, Split, SplitItem, Text, TextVariants } from '@patternfly/react-core';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Split,
+  SplitItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
 
 type OverviewCardProps = {
   obj?: V1beta1ForkliftController;
   loaded?: boolean;
   loadError?: unknown;
+  onHide?: () => void;
 };
 
-export const OverviewCard: FC<OverviewCardProps> = () => {
+export const OverviewCard: FC<OverviewCardProps> = ({ onHide }) => {
   const { t } = useForkliftTranslation();
+  const actionDropdownItem: any[] = [];
 
+  const [menuIsOpen, setMenuIsOpen] = React.useState(false);
+  const onToggle = () => setMenuIsOpen((open) => !open);
+
+  if (onHide) {
+    actionDropdownItem.push(
+      <DropdownItem
+        key="action"
+        component="button"
+        description={t(
+          "You can always bring this welcome card back into view by clicking 'Show the welcome card' in the page heading.",
+        )}
+        onClick={onHide}
+        data-test="hide"
+        style={{ whiteSpace: 'pre-wrap', width: 280, fontWeight: 'bold' }}
+      >
+        {t('Hide from view')}
+      </DropdownItem>,
+    );
+  }
   return (
     <Card>
+      <CardHeader>
+        <CardActions>
+          <Dropdown
+            isOpen={menuIsOpen}
+            isPlain
+            toggle={<KebabToggle onToggle={onToggle} data-test="actions" />}
```

Comment:
```suggestion
            toggle={<KebabToggle onToggle={onToggle} data-testid="actions" />}
```
in our code we use the default `data-testid` arg name
https://testing-library.com/docs/queries/bytestid/#options

Reply:
I see that there are few other places in our code that use data-test attribute as well (new code, not legacy) so worth fixing those as well in a separate PR. 
For now, I'll fix the occurrences relevant to this fix only.

Reply:
> so worth fixing those as well in a separate PR.

great ! , i'll make an issue for that

Reply:
Ref: https://github.com/kubev2v/forklift-console-plugin/issues/837